master/internal/trials/api_trials.go

```diff
@@ -338,3 +345,55 @@ func (a *TrialsAPIServer) DeleteTrialsCollection(
 
 	return &apiv1.DeleteTrialsCollectionResponse{}, nil
 }
+
+// ValidationMetricsSeries returns a time-series of the specified validation metric in the specified
+// trial.
+func ValidationMetricsSeries(trialID int32, startTime time.Time,
```

Comment:
This is no longer a function reliant on `pgDb` so it did not seem to make sense to be there, but if you disagree I am happy to move it back.

Reply:
consider a perspective of someone unfamiliar with the code and working with metrics. discovering these two in different files would be a surprise in my opinion.

however as I mentioned in another comment, it doesn't make sense to be not to support training metrics, so if we add them, this issue may get naturally resolved.

also if we were to move these, I'd put it into `internal/trials/postgres_trials.go` because it's a database thing (or model thing, in MVC sense).